layouts/services/single.html

```diff
@@ -69,17 +69,17 @@ <h2 class="mb-4 font-bold text-xl sm:text-2xl lg:text-3xl">{{ .title }}</h2>
             <p class="md:text-xl">{{ .subtitle }}</p>
             <div class="grid md:grid-cols-2 lg:grid-cols-3 gap-8 mt-8 text-center">
                 {{ range .studies }}
-                <div class="bg-[#1D65A6] pb-4 rounded-md text-[#FCFCFC]">
+                <div class="bg-[#E3E3E1] pb-4 rounded-md">
                     <div>
                         <h4 class="text-lg md:text-xl font-bold my-6">{{ .client_name}}</h4>
-                        <p class="md:text-lg mx-4 pb-4 md:pb-0 md:h-72 xl:h-64">{{ .summary }}</p>
+                        <p class="md:text-lg mx-4 pb-8">{{ .summary }}</p>
                     </div>
                     <a href="{{ .link }}" class="flex justify-center items-center md:text-lg font-bold">
                         <span class="mr-1">
                             Learn More 
                         </span>
-                        <img src="img/arrow-right.png"/>
-                        <img src="img/arrow-right.png" />
+                        <i class="fa fa-chevron-right"></i>
```

Comment:
The image was replaced with an icon to ensure the color always matches the text color. This will prevent us from having to change the image any time the text color changes.